packages/app/src/components/ProviderFields.tsx

```diff
@@ -133,7 +134,7 @@ export const ProviderFields: FC<ProviderFieldsProps> = ({ buttonText, control, e
         <Controller
           control={control}
           name="email"
-          render={({ field }) => <TextField {...field} label="Email Address" variant="outlined" />}
+          render={({ field }) => <TextField {...field} disabled={true} label="Email Address" variant="outlined" />}
```

Comment:
For booleans you just have to provide the name, so you could just say `disabled`